mon-pix/app/controllers/authentication/login-or-register-oidc.js

```diff
@@ -27,6 +30,14 @@ export default class LoginOrRegisterOidcController extends Controller {
     return this.url.showcase;
   }
 
+  get oidcUserAuthenticationStorage() {
+    return oidcUserAuthenticationStorage.get();
+  }
+
+  get userClaims() {
+    return this.oidcUserAuthenticationStorage?.userClaims ?? null;
```

Comment:
L'operateur [`.?`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining renvoie déjà un _Falsy_, pas besoin de chercher à renvoyer `null` :
) 

```suggestion
    return this.oidcUserAuthenticationStorage?.userClaims;
```